plugins/module_utils/linode_common.py

```diff
@@ -104,7 +104,7 @@
 RETRY_STATUSES = {408, 429, 502}
 
 
-class LinodeModuleBase:
```

Comment:
Did pylint complain about this?

Reply:
I also included the lint fix in Makefile (#592)